docs/orchestration/concepts/kv_store.md

```diff
@@ -0,0 +1,123 @@
+# KV Store
+
+Key Value Store is your managed metadata database within Prefect Cloud.
+
+**Keys** are strings. **Values** are json blobs.
+
+The number of key value pairs allowed is limited by license, starting with 50 pairs on the Free tier. Values are limited to 1 MB in size.
```

Comment:
Should we will list out the number of pairs for each license here?

Reply:
Hmm I stayed away from listing out every tier because it could change in the future. And it would make for a frustrating Sales process if our Core docs + backend + Sales team aren't all in sync.

Reply:
Ahh okay. That makes sense.